Table.ExpandListColumn -- documenting behavior on nested table columns (#171)
MicrosoftDocs/query-docs: query-languages/m/table-expandlistcolumn.md

  
## About

> Given a `table`, when `column` contains a list of values, splits the list into a row for each value. If, instead, `column` contains a nested table, treats the nested table as though it were a list of records, which it then splits into a row for each value. Values in the other columns are duplicated in each new row created.

Reviewer: ```suggestion
Given a `table` where `column` contains a list of values, splits the list into a row for each value. Values in the other columns are duplicated in each new row created. This function can also expand nested tables by treating them as lists of records.
```

)
```

## Example 2

> Split the nested table column [Components] in the table.

Reviewer: Nix "in the table", both here and above in the existing example.

```suggestion
Split the nested table column [Components].
```

Author: Good idea. Done.
